services/src/main/resources/org/keycloak/protocol/oidc/endpoints/3p-cookies-step1.html

```diff
@@ -5,15 +5,20 @@
   </head>
   <body>
     <script type="module">
-      // Check if the browser has granted us access to 3rd-party storage (such as cookies).
-      const hasAccess = await hasStorageAccess();
+      attemptCookie();
+      
+     // Encapsulate the code in an async to avoid syntax error
```

Comment:
I don't think we need a comment for this, this is also exclusive to older versions of certain browsers.

```suggestion
```